.gitignore

```diff
@@ -6,3 +6,4 @@
 /test/allow
 /test/config/direnv/allow
 *.sw?
+/.idea
```

Comment:
I don't mind it but just so you know, this is usually better handled in a [global gitignore](https://stackoverflow.com/questions/7335420/global-git-ignore#7335487) as it's unrelated to the project.